parsil/src/executor.rs

```diff
@@ -289,30 +292,116 @@ fn convert_funcalls(expr: &mut Expr) -> Result<()> {
     Ok(())
 }
 
-fn expand_block_range<C: ContextProvider>(settings: &ParsilSettings<C>) -> Expr {
-    funcall(
-        "generate_series",
-        vec![
-            funcall(
-                "GREATEST",
-                vec![
-                    Expr::Identifier(Ident::new(VALID_FROM)),
-                    Expr::Value(Value::Placeholder(
-                        settings.placeholders.min_block_placeholder.to_owned(),
-                    )),
-                ],
-            ),
-            funcall(
-                "LEAST",
-                vec![
-                    Expr::Identifier(Ident::new(VALID_UNTIL)),
-                    Expr::Value(Value::Placeholder(
-                        settings.placeholders.max_block_placeholder.to_owned(),
-                    )),
-                ],
-            ),
-        ],
-    )
+// Build the subquery that will be used as the source of epochs and block numbers
```

Comment:
Use `///` here instead of `//`

Reply:
Done in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/425/commits/23affb5f3b0c62fc1ab825043196a2f89e20039c